CONTRIBUTING.md

```diff
@@ -128,45 +159,11 @@ del(src)
 ```
 ***
 
-Do not return unused values from functions. Do not use return if there's no any actions in function after it. 
-
-***Good:***
-```
-proc/mutate_count(var/obj, var/value)
-	if(!value)
-	    return
-    obj.count = value
-```
-***Bad:***
-```
-proc/mutate_count(var/obj, var/value)
-	obj.count = value
-	return 1
-```
-***
-
-Instance variables should be used with only with source object denotation to avoid namespace conflicts.
```

Comment:
This is telling you not to use `src` arbitrarily and only use it when you have two conflicting variable names. Ideally you would probably name it something else, but this is also sound advice and tells people the only acceptable instance of `src`. Unless you're going to re-write this to offer similar advice about using `src`, I do not think this should be deleted.

Reply:
It doesn't say that at all. Infact i'm pretty sure the above line is telling you to always use src.thing

I don't see where your assertion comes from, what do you think instance variables means?

Reply:
> Instance variables should be used with only with source object denotation to avoid namespace conflicts.

> only with source object denotation to avoid namespace conflicts.

> only with

>  **only**

Reply:
Yes, but what does instance variables mean?
Function parameters are not instance variables. 

An instance is an object or datum, a thing that exists in the world, a practical application of code.
It's saying all variables belonging to this instance should use src. IE, use src.thing every time you refer to something on yourself

It doesn't say to do this in the case of a conflict, it's telling you to do it always to prevent any possibility of a conflict from occurring

There's no need for that, its inefficient and wasteful

Reply:
but moreover, theres no need for it even with function inputs. As i just told sahqo, if you use src in a function you have to keep using it for the rest of that context, its adding pointless labour to the user and introducing points of failure if you slip up. 

If you name your input parameters properly, thats exactly one point where you have to fix things. It's more foolproof